### PR TITLE
Add trading symbol to position metrics

### PR DESCRIPTION
**Pull request by bonomat**
it's prepared to dea with multiple trading pairs. The idea is to sort the map of CFDs by trading pair first and then pass down to `update_position_metrics`.

prepares https://github.com/itchysats/itchysats/issues/2464

### CODE REVIEW

**Review by da-kami — APPROVED** [review]

File: daemon/src/position_metrics.rs [hunk 1]
```diff
@@ -377,30 +382,35 @@ mod metrics {
     const STATUS_REJECTED_LABEL: &str = "rejected";
     const STATUS_REFUNDED_LABEL: &str = "refunded";
 
+    const SYMBOL_LABEL: &str = "symbol";
+    pub const BTCUSD_LABEL: &str = "BTCUSD";
```
Comment (klochowicz) [inline daemon/src/position_metrics.rs:386]: maybe for consistency with broader financial system we could use `XBTUSD`?

**Comment by klochowicz** [comment]
I'm not that familiar with metrics, but I'm not sure whether this is the best practice.
wouldn't this add a lot of data as this string is now appended to all of these passed metrics? how would this affect our metrics ingestion rate? 

I guess the alternative would be to expose a metric per trading pair? (maybe in a hashmap?)

**Comment by bonomat** [comment]
Long answer: 

It is best practice but it will indeed increase the number of metrics, i.e. we will have a metric for each trading pair. 
The number of `symbols` should be small (2 to be precise), so it does not grow too much. 

Short answer: 
There is no alternative :D 

Fun fact: 
```
metric_1{symbol='btcusd'}
metric_1{symbol='ethusd'}
```
has the same count as: 
```
metric_1_btc_usd
metric_1_eth_usd
```

The former one is the one to prefer as it would allow you to easily do aggregates over all `symbol`s.

File: daemon/src/position_metrics.rs (hunk 1 above)
Comment (bonomat) [reply daemon/src/position_metrics.rs:386]: I think it's pretty random if exchange use `xbtusd` or `btcusd`

**Pull request merged**